### DISCUSSION TITLE
CTRL+SHIFT+E (Open Event Log) will make screen shot when Radeon Software is installed

### DISCUSSION THREAD

**Discussion by Ageless93** [BOINC/boinc#5055] [Q&A]
I found tonight when trying to open the Event Log from BOINC Manager that each time I pressed CTRL+SHIFT+E simultaneously my AMD Radeon Software made a screen shot. At least CTRL+SHIFT+S is also used by this software and it takes precedence over BOINC. 

I'm not sure how this should be fixed, but I think we should at least put a warning out for AMD users who update to the new Radeon Software 20.1.1 or above, as they'll be affected in this way. 

Screen shot: https://photos.app.goo.gl/63xbTrTPvNdW2j5q6

**Comment by AenBleidd** [comment]
@Ageless93, are these shortcuts in AMD software hardcoded? If not - there will be too hard or even not possible to find out whether these shortcuts are binded by someone or not

**Comment by Ageless93** [comment]
It's not hardcoded, I can change the shortcuts or even remove them. Trouble is that they come default at those settings for anyone installing the software and - I think because this software gets added to the PATH - it always takes precedence over any keyboard shortcuts in BOINC Manager, even with BOINC Manager open and it being the app in focus.

**Comment by AenBleidd** [comment]
Because global shortcuts have higher priority than local one.
I'm not sure this is smth we should handle

**Comment by Ageless93** [comment]
Well, it's perhaps not something we should try to fix, but a warning in the User Manual documentation ought to be possible. I'm just wondering if it only does this under Windows?
